events/job_offmap.txt

```diff
@@ -1168,8 +1213,11 @@ character_event = {
 						factor = 0
 						OR = {
 							trait = light_foot_leader
-							trait = heavy_infantry_leader 
```

Comment:
Put heavy_infantry_leader to its original place so that the GitHub will stop thinking that you removed it.